src/styles/index.scss

```
@@ -1,3 +1,87 @@
 body {
   margin: 0;
+  position: relative;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+}
+
+.stopwatch {
+  width: 80vmin;
+  height: 80vmin;
+
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+  border: 1vmin dotted #000;
+  border-radius: 50%;
+
+  &-hands {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    position: relative;
+    width: 100%;
+    height: 100%;
+
+    &-minutes {
+      height: 20vmin;
+      width: 3vmin;
+      background-color: #0700ff;
+      transform-origin: 50% 100%;
+      animation: rotate-minute 3600s steps(60, end) infinite;
+
+      position: absolute;
+      top: 20vmin;
+    }
+
+    &-seconds {
+      height: 38vmin;
+      width: 1.5vmin;
+      background-color: #2c8000;
+      transform-origin: 50% 100%;
+      animation: rotate-second 60s linear 1s infinite;
+
+      position: absolute;
+      top: 2vmin;
+    }
+  }
+
+  &-center {
+    width: 5vmin;
+    height: 5vmin;
+    background-color: #f6a603;
+    border-radius: 50%;
+    z-index: 1;
+
+    position: absolute;
+  }
+}
+
+@keyframes rotate-second {
+  from {
+    transform: rotate(0deg);
+  }
+  to {
+    transform: rotate(360deg);
+  }
+}
+
+@keyframes rotate-minute {
+  from {
+    transform: rotate(0deg);
+  }
+  to {
+    transform: rotate(360deg);
+  }
+}
```

it is the same action, so use only 1 `rotate`
```suggestion
@keyframes rotate {
  from {
    transform: rotate(0deg);
  }
  to {
    transform: rotate(360deg);
  }
}
```